test/syncenginetestutils.h

```diff
@@ -529,7 +531,10 @@ class FakeFolder
     [[nodiscard]] OCC::SyncEngine &syncEngine() const { return *_syncEngine; }
     [[nodiscard]] OCC::SyncJournalDb &syncJournal() const { return *_journalDb; }
 
-    FileModifier &localModifier() { return _localModifier; }
+    DiskFileModifier &localModifier()
+    {
+        return _localModifier;
+    }
```

Comment:
breaks style and compilation

Reply:
Apologies, removed this change and fixed the test